runtime.txt

```diff
@@ -0,0 +1 @@
+python-3.6
```

Comment:
What is this file ? Is it possible to hide it at least (.runtime.txt ?) if it is really necessary ?